integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -68,11 +68,39 @@ protected static String getDeleteSql(String tableName, String condition) {
     return String.format("DELETE FROM %s where %s", tableName, condition);
   }
 
+  private static String getUpdateTableSql(String tableName, String setClause, String whereClause) {
+    return String.format("UPDATE %s SET %s WHERE %s", tableName, setClause, whereClause);
+  }
+
+  private static String getRowLevelUpdateTableSql(
```

Comment:
If it's used for Iceberg only, we could place it in IcebergIT

Reply:
paimon will use them when we support paimon catalog in spark-connector.

Reply:
ok